include/zephyr/dt-bindings/clock/stm32f0_clock.h

```diff
@@ -73,5 +73,7 @@
 #define USART3_SEL(val)		STM32_CLOCK(val, 3, 18, CFGR3_REG)
 /** BDCR devices */
 #define RTC_SEL(val)		STM32_CLOCK(val, 3, 8, BDCR_REG)
+/** Dummy: Add a specificier when no selection is possible */
+#define NO_SEL			0xFF
```

Comment:
could   `NO_SEL`  be defined only _once_ in the #include <zephyr/drivers/clock_control/stm32_clock_control.h>

Reply:
Not possible. `drivers/clock_control/stm32_clock_control.h` is not a device tree binding (could not be included in *.dtsi)